src/renderer/components/modals/OrderConfirmationModal.vue

```diff
@@ -198,6 +198,11 @@ export default {
     .offer {
       margin-bottom: $space-xs;
     }
+
+    .taking > div:first-child {
+      overflow-y:auto;
+      max-height: toRem(400px);
```

Comment:
is there a way we can ensure on a short screen that this won't push the buttons beyond the bottom of the screen? Suppose they only have a total 768 vertical pixels. If we want to keep this fixed we should actually probably set it to 300px